test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14218,14 +14218,25 @@ def create_subjob_id(self, job_array_id, subjob_index):
         return job_array_id[:idx + 1] + str(subjob_index) + \
             job_array_id[idx + 1:]
 
-    def create_eatcpu_job(self, duration=None):
+    def create_eatcpu_job(self, duration=None, mom=None):
         """
         Create a job that eats cpu indefinitely or for the given
         duration of time
         """
+        if self.du is None:
```

Comment:
you don't need these 2 lines as I just checked Job class has `self.du` already defined as a class variable and it will never be None

Reply:
Hi Hiren, 
At first I tried self.du but i was getting AttributeError as:
self.du.run_copy(hosts=mom, src=script_path, dest=d)
AttributeError: 'NoneType' object has no attribute 'run_copy'
That is the reason I had to add these two lines here